ood-portal-generator/spec/fixtures/ood-portal.conf.all

```diff
@@ -195,6 +195,26 @@ Listen 8080
     LuaHookLog analytics.lua analytics_handler
   </Location>
 
+  # CORS setup for API requests
+  <Location "/pun/test">
+    Header always set Access-Control-Allow-Origin "test:8080"
+    Header always set Access-Control-Allow-Methods "POST, GET, OPTIONS, DELETE, PUT"
+    Header always set Access-Control-Allow-Headers "x-requested-with, content-type, origin, authorization, accept, client-security-token"
+    Header always set Access-Control-Expose-Headers "Content-Security-Policy, Location"
+    Header always set Access-Control-Allow-Credentials "false"
+    Header always set Access-Control-Max-Age "600"
+
+    # CORS PREFLIGHT REQUESTS ARE NOT AUTHENTICATED.
+    <Limit OPTIONS>
+      allow from all
+      Satisfy Any
+    </Limit>
+
+    RewriteEngine On
+    RewriteCond %{REQUEST_METHOD} OPTIONS
+    RewriteRule ^(.*)$ $1 [R=204,L]
```

Comment:
I wonder if this redirect is right. If I send an OPTIONS preflight request isn't it going to redirect me to a GET page. Let's say I call OPTIONS on `/api/v1/batch_connect/sessions`. Wouldn't this redirect actually give me the content of sessions without having to authenticate?

Which now that I look here, shouldn't this Location have the `Auth` settings? I think at a minimum we'd have to launch with SSO, meaning this location has the same settings as the other Locations.

Reply:
Got it! You're right. I'm used to using it for 301 or 302 redirects. We'll definitely need test cases around preflights to make sure they work but other paths need authentication.